examples/pipeline/README.md

```diff
@@ -85,7 +83,7 @@ git submodule add https://github.com/primeharbor/pht-account-configurator
 git submodule init
 git submodule update
 
-EOF
+<!--EOF
```

Comment:
Since the below seemed to be a note to self, I figured not rendering it would make sense. Up to you